src/main/java/org/zalando/nakadi/service/subscription/SubscriptionService.java

```diff
@@ -105,6 +111,14 @@ public SubscriptionService(final SubscriptionDbRepository subscriptionRepository
                 .map(Optional::get)
                 .forEach(eventType -> client.checkScopes(eventType.getReadScopes()));
 
+        if (featureToggleService.isFeatureEnabled(DISABLE_SUBSCRIPTION_CREATION)) {
+            try {
+                return Result.ok(getExistingSubscription(subscriptionBase));
```

Comment:
if you just return existing subscription here - the user will get a wrong response code 201, like it was really created.